src/app/content/components/Page/PageComponent.tsx

```diff
@@ -47,6 +47,19 @@ export default class PageComponent extends Component<PagePropTypes> {
 
       transformContent(parsedContent, parsedContent.body, this.props.intl);
 
+      if (this.props.lockNavigation) {
```

Comment:
i see why this needs to be here, because in `transformContent` you don't have access to the props to perform the mutation based on the `lockNavigation` flag, but can you move the content of this if statement into a function in https://github.com/openstax/rex-web/blob/ecb62adf6d2bad690953b5b31a3665d5f5058e23/src/app/content/components/Page/contentDOMTransformations.ts and just call it from here? that way it reduces visible complexity in this file